test/testcondition.cpp

```diff
@@ -645,7 +645,7 @@ class TestCondition : public TestFixture {
               "}");
         ASSERT_EQUALS("", errout_str());
 
-        check("void f(int i) {\n"
+        check("void f(int64_t i) {\n"
```

Comment:
The test was added in https://github.com/danmar/cppcheck/commit/8ea5df62c4bfe882a2e4d825260c4b0561e7ce65, so it is unrelated to `compareValueOutOfTypeRangeError`.